tabs/src/ui-components/PageLayout.tsx

```diff
@@ -0,0 +1,14 @@
+
+
+type Props = {
+    children?: React.ReactNode;
+};
+
+export const PageLayout: React.FC<Props> = ({children}) => {
+    return (
+        <>
```

Comment:
What is this `<>`?

Reply:
It shows all the children items. In the components pages. Meaning this is the master pageout. and everything else falls inbetween it. So we add the Nav/Footer component here and then it displays on all pages, without having to add the nav/footer component on every page.